scaladoc/src/dotty/tools/scaladoc/Scaladoc.scala

```diff
@@ -45,7 +45,8 @@ object Scaladoc:
     apiSubdirectory : Boolean = false,
     scastieConfiguration: String = "",
     defaultTemplate: Option[String] = None,
-    quickLinks: List[QuickLink] = List.empty
+    quickLinks: List[QuickLink] = List.empty,
+    dynamicSideMenu: Boolean = false,
```

Comment:
I think it should be dynamic as a default for now (so the default behavior doesn't change). We can change it to default after we ensure it works in practice.

Reply:
`false` is the old behavior

Reply:
Oh, right, this one `if` confused me. So everything looks alright!